src/main/java/com/puppycrawl/tools/checkstyle/Checker.java

```diff
@@ -66,7 +66,7 @@
     public static final String EXCEPTION_MSG = "general.exception";
 
     /** Logger for Checker. */
-    private static final Log LOG = LogFactory.getLog(Checker.class);
+    private final Log log;
```

Comment:
Then there are no other classes with private static final definition for loggers. Looks like there wasn't an issue after all